Domain/SecurityIdentityRetrievalStrategy.php

```diff
@@ -75,4 +84,24 @@ public function getSecurityIdentities(TokenInterface $token)
 
         return $sids;
     }
+
+    /**
+     * Symfony < 4.3 BC layer.
+     *
+     * @param TokenInterface $token
+     *
+     * @return array
+     */
```

Comment:
I'd suggest removing the full dobclock here, it doesn't provide enough value.

Reply:
@nicolas-grekas Done.